lib/explorer/data_frame.ex

```diff
@@ -648,6 +648,7 @@ defmodule Explorer.DataFrame do
   This collects the lazy data frame into an eager one, computing the query.
 
   If already eager, this is a noop.
+  Collecting a grouped dataframe should return a grouped dataframe.
```

Comment:
Just FYI, this will show up as a single paragraph in docs. If you want a separate paragraph, you need two newlines:

```suggestion

  Collecting a grouped dataframe should return a grouped dataframe.
```

The same applies to examples below.